src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs.jsx

```diff
@@ -70,6 +70,36 @@ const BreadcrumbsInterface = React.createClass({
     }).reverse(); // un-reverse rendered result
   },
 
+  getVirtualCrumb() {
+    let evt = this.props.event;
+    let crumb;
+
+    let exception = evt.entries.find(entry => entry.type === 'exception');
+    if (exception) {
+      let {type, value, module} = exception.data.values[0];
+      crumb = {
+        type: 'error',
+        level: 'error',
+        category: moduleToCategory(module || null) || 'exception',
+        data: {
+          type: type,
+          value: value
+        }
+      };
+    } else if (evt.message) {
+      let levelTag = evt.tags.find(t => t.key === 'level');
+      let level = levelTag && levelTag.value;
+      crumb = {
+        type: 'message',
+        level: level,
+        category: 'message',
+        message: evt.message
+      };
+    }
+    crumb.timestamp = evt.dateCreated;
```

Comment:
it seems like it's possible for `crumb` to be undefined, so we should probably check for that before trying to set `crumb.timestamp`